### PR TITLE
Aastha_Assignment_29_Done_30mins

### PR DESCRIPTION
**Pull request by AasthaJJun20**
-> pushed class InvertTheCaseOfArrayElements for assignment-29
->PFB screenshot of output:
InvertTheCaseOfArrayElements-output

### CODE REVIEW

**Review by NikhilMJun20 — APPROVED** [review]
Coding logic looks good to me.
I am approving the PR.

File: src/aasthaJJun20/InvertTheCaseOfArrayElements.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Output:
+array = {"aAvRUTI", "nIkHIL","aDItI", "maHESH","aRCHAna","JAgaDEESH"};*/
+
+public class InvertTheCaseOfArrayElements {
+	//Below method displays the array elements on console
+	void diplayArrayElements(String[] input) {
+		System.out.print("{  ");
+		for (int index = 0; index < input.length; index++)
+			System.out.print(input[index] + "  ");
```
Comment (KrishnaKTechnocredits) [inline src/aasthaJJun20/InvertTheCaseOfArrayElements.java:15]: You can directly use  Arrays.toString(input) this method.

File: src/aasthaJJun20/InvertTheCaseOfArrayElements.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+		System.out.print("}");
+	}
+	
+	//Below method converts the case of every character of every array element
+	void invertCase(String[] input) {
+		System.out.print("Input array : ");
+		diplayArrayElements(input);
```
Comment (KrishnaKTechnocredits) [inline src/aasthaJJun20/InvertTheCaseOfArrayElements.java:22]: Arrays.toString(input) Just need to write this line. No need to of diplayArrayElements() method.

Comment (AasthaJJun20) [reply src/aasthaJJun20/InvertTheCaseOfArrayElements.java:22]: Got it Krishna!! Thank you!!

**Pull request merged**